### DISCUSSION TITLE
Want to delete a key by ETCD API v3.4

### DISCUSSION THREAD

**Discussion by Tridu33** [etcd-io/etcd#18544] [Q&A] [answered]
### What would you like to be added?

can not find "How to delete a key by ETCD API v3.4" in https://etcd.io/docs/v3.1/learning/api/ .

### Why is this needed?

need delete key by API but not `./etcdctl del balabala`

**Comment by ivanvc** [comment, marked as answer]
Hi @Tridu33, please refer to v3.4 documentation: https://etcd.io/docs/v3.4/learning/api/#delete-range
